third_party/go/BUILD

```diff
@@ -61,24 +75,66 @@ go_get(
 )
 
 go_get(
-    name = 'context',
-    get = 'golang.org/x/net/context',
-    revision = '57efc9c3d9f91fb3277f8da1cff370539c4d3dc5',
+    name = 'net',
+    get = 'golang.org/x/net/...',
+    revision = '136a25c244d3019482a795d728110278d6ba09a4',
+    deps = [
+        ':text',
+        ':terminal',
+    ],
+)
+
+go_get(
+    name = 'text',
+    get = 'golang.org/x/text/...',
+    revision = '4e4a3210bb54bb31f6ab2cdca2edcc0b50c420c1',
 )
 
 go_get(
     name = 'grpc',
-    exported_deps = [':context'],
+    exported_deps = [':net'],
     get = 'google.golang.org/grpc',
-    install = ['google.golang.org/grpc/health'],
+    install = [
+        'google.golang.org/grpc/balancer',
```

Comment:
This is a bit of a mouthful - would it be easier to use `get = 'google.golang.org/grpc/...'`?

Reply:
I tried that first. 😔 It fails, at least on Go 1.9, because one of the sub packages contains only tests; the way the go_get rule effectively calls 'go tool compile *.go' in that directory leads to a fatal error.

Reply:
Ah, right. Not much to be done then I guess :)